src/lib/y2storage/dialogs/guided_setup/select_root_disk.rb

```diff
@@ -161,6 +161,43 @@ def update_settings!
           update_windows_settings if activate_windows_actions?
         end
 
+        def help_text
+          # TRANSLATORS: Help text for root disk selection
+          msg = _(
+            "<p>" \
+            "Select the disk where to create the root filesystem. " \
+            "</p><p>" \
+            "This is also the disk where any boot-related partitions " \
+            "will be created as necessary: /boot, ESP (EFI System " \
+            "Partition), BIOS-Grub. " \
```

Comment:
Not sure if we can say that any boot-related partition will be created there.

The point is that we consider that the BIOS / firmware will use that disk for booting, which doesn't always imply we will create the boot-related partitions there (at least not in all booting systems).

Reply:
I fear using a 100% correct wording here would pretty much defeat the purpose of being a help text because it would quickly become too complicated.

I think I'll cheat my way around that by adding _typically_.

Reply:
See also PR#853

Reply:
Yes, much better with "typically". :-)